src/docbuilder/rustwide_builder.rs

```diff
@@ -1218,4 +1232,13 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    #[ignore]
+    fn test_build_std() {
+        wrapper(|env| {
+            assert!(RustwideBuilder::init(env)?.build_local_package(Path::new("tests/build-std"))?);
```

Comment:
nit: should we name this folder `test-crates` or `tests/crates/` to make it clear that it's crates used by tests?